src/components/TodoComponents/Instance.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+
+const Instance = props => {
```

Comment:
Why rename this component to `Instance` instead of retaining its original name? You could also just rename it to `TodoItem` or `TodoInstance` just to be more specific/declarative.